services/common/src/components/projectSummary/AuthorizationSupportDocumentUpload.tsx

```diff
@@ -34,9 +31,7 @@ export const AuthorizationSupportDocumentUpload: FC<AuthorizationSupportDocument
     const newDocuments = documents.filter(
       (file) => fileToRemove.serverId !== file.document_manager_guid
     );
-    newDocuments.forEach((newDoc) => {
-      updateAmendmentDocuments(newDoc, code);
-    });
+    updateAmendmentDocuments(newDocuments);
```

Comment:
the changes to updateAmendmentDocuments is a fix for the bug Alex posted in the bugs channel today. Before it was updating a single document, not really having the power of _removing_ documents. Now it takes as its parameter the desired value of the field, allowing us to remove documents here.